ai/mistralai/v0/config/tasks.json

```diff
@@ -0,0 +1,429 @@
+{
+  "$defs": {
+    "multi-modal-content": {
+      "instillFormat": "structured/multi-modal-content",
+      "items": {
+        "properties": {
+          "image-url": {
+            "properties": {
+              "url": {
+                "description": "Either a URL of the image or the base64 encoded image data.",
+                "type": "string"
+              }
+            },
+            "required": [
+              "url"
+            ],
+            "type": "object"
+          },
+          "text": {
+            "description": "The text content.",
+            "instillFormat": "string",
+            "type": "string"
+          },
+          "type": {
+            "description": "The type of the content part.",
+            "enum": [
+              "text",
+              "image_url"
+            ],
+            "instillFormat": "string",
+            "type": "string"
+          }
+        },
+        "required": [
+          "type"
+        ],
+        "type": "object"
+      },
+      "type": "array"
+    },
+    "chat-message": {
+      "properties": {
+        "content": {
+          "$ref": "#/$defs/multi-modal-content",
+          "description": "The message content",
+          "instillUIOrder": 1,
+          "title": "Content"
+        },
+        "role": {
+          "description": "The message role, i.e. 'system', 'user' or 'assistant'",
+          "instillFormat": "string",
+          "instillUIOrder": 0,
+          "title": "Role",
+          "type": "string"
+        }
+      },
+      "required": [
+        "role",
+        "content"
+      ],
+      "title": "Chat Message",
+      "type": "object"
+    },
+    "chat-usage": {
+      "description": "Token usage on the Mistral platform text generation models",
+      "instillUIOrder": 1,
+      "properties": {
+        "input-tokens": {
+          "description": "The input tokens used by Mistral models",
+          "instillFormat": "number",
+          "instillUIOrder": 2,
+          "title": "Input Tokens",
+          "type": "number"
+        },
+        "output-tokens": {
+          "description": "The output tokens generated by Mistral models",
+          "instillFormat": "number",
+          "instillUIOrder": 3,
+          "title": "Output Tokens",
+          "type": "number"
+        }
+      },
+      "required": [
+        "input-tokens",
+        "output-tokens"
+      ],
+      "title": "Usage",
+      "type": "object"
+    },
+    "embedding-usage": {
+      "description": "Token usage on the Mistral platform embedding models",
+      "instillUIOrder": 1,
+      "properties": {
+        "tokens": {
```

Comment:
Question
which token do you use here?
Is prompt_token always same as total_token?

Please help us confirm it!
Thank you!
[Doc](https://docs.mistral.ai/api/#operation/createEmbedding)
<img width="541" alt="image" src="https://github.com/instill-ai/component/assets/64747455/c86c13f7-cf23-4dec-b47f-b11edba7d383">

Reply:
Ah, I got it.
It seems the total one is for the functions with the multiple types of tokens.
And for embedding, it only has 1 type.

Ok for me here!

Reply:
Great!